components/sst_importer/src/sst_importer.rs

```diff
@@ -202,25 +212,29 @@ impl SSTImporter {
 
         let range_start = meta.get_range().get_start();
         let range_end = meta.get_range().get_end();
+        let range_start_bound = key_to_bound(range_start);
+        let range_end_bound = if is_raw_kv && meta.get_end_key_exclusive() {
```

Comment:
So the current non-raw situation is `is_raw_kv = false, end_key_exclusive = false`. 

This PR is intended to serve `is_raw_kv = true, end_key_exclusive = true`. 

1. Is there any case where `is_raw_kv = true, end_key_exclusive = false` is possible? If no, the `is_raw_kv &&` part is unnecessary 
2. does this function has any other use case of `is_raw_kv`? If no, the entire parameter is redundant and could be reverted.